index.js

```diff
@@ -285,7 +284,6 @@ module.exports = {
       },
     ],
     'sort-keys': 'off',
-    'sort-keys-fix/sort-keys-fix': 'warn',
```

Comment:
Didn't know there was a replacement 😮 

What other rule covers this?

Reply:
`typescript-sort-keys`

Reply:
Previously, `typescript-sort-keys` didn't offer an autofix